src/contents/en-gb/doc_dotnet.md

```diff
@@ -0,0 +1,265 @@
+# SAFE .Net Desktop App Tutorial
+
+In this tutorial, we will create a [.Net Framework](https://docs.microsoft.com/en-us/dotnet/framework/) console application. The .NET Framework is a development platform for building apps for web, Windows, Windows Phone, Windows Server, and Microsoft Azure using C#, Visual Basic and F#. And, this tutorial can also be used for developing [.Net Core](https://github.com/dotnet/core) apps.

```

Comment:
Remove `And` from the beginning of the last sentence

Reply:
removed